packages/go/ein/ad.go

```diff
@@ -278,6 +302,28 @@ func ParsePrimaryGroup(item IngestBase, itemType graph.Kind, primaryGroupSid str
 	return NewIngestibleRelationship(IngestibleSource{}, IngestibleTarget{}, IngestibleRel{})
 }
 
+func ParseDomainForIdentity(item IngestBase, itemType graph.Kind, domainSID string) IngestibleRelationship {
+	if domainSID != "" {
```

Comment:
Nit picky, but I would think an early exit condition on the unhappy path would make more sense

```
if domainSid == "" {
    return NewIngestibleRelationship(IngestibleSource{}, IngestibleTarget{}, IngestibleRel{})
} else {
 ....
}
```

Although it looks like the `TODO:` may have played a factor into this decision